### ISSUE TITLE
FATA[0001] - repository does not exist when --json-legacy flag is passed

### ISSUE THREAD

**Issue by bugbaba** [trufflesecurity/trufflehog#419]
Hello :)

**Describe the bug**
When `--json-legacy` flag is passed the application exits with `FATA[0001] could open repo:                              error="repository does not exist"
` error message while its working perfectly when `--json` flag is used instead.

**To Reproduce**
Steps to reproduce the behaviour:
1. Download the latest linux binary version `3.3.1`
2. run `./trufflehog github --repo https://github.com/bugbaba/test4truffleHog --json --debug` the program runs successfully.
3. run `./trufflehog github --repo https://github.com/bugbaba/test4truffleHog --json-legacy --debug` and the application run into error.

**Expected behavior**
Application should not crash when `--json-legacy` is passed and return output in that format.

**Output**
```
» ./trufflehog github --repo https://github.com/bugbaba/test4truffleHog --json-legacy --debug
DEBU[0000] running version 3.3.1                        
2022/04/16 15:23:02 [updater parent] run
2022/04/16 15:23:03 [updater parent] checking for updates...
2022/04/16 15:23:03 [updater parent] failed to get latest version: already up to date
2022/04/16 15:23:03 [updater parent] starting /home/kyle/Downloads/trufflehog
DEBU[0000] running version 3.3.1                        
2022/04/16 15:23:03 [updater child#1] run
2022/04/16 15:23:03 [updater child#1] start program
trufflehog 3.3.1
DEBU[0000] running with up to 6 workers                 
DEBU[0000] loaded 2 decoders                            
DEBU[0000] loaded 631 detectors total, 631 with verification enabled. 0 with verification disabled 
INFO[0000] starting pprof and fgprof server on :18066 /debug/pprof and /debug/fgprof 
DEBU[0000] Found 2 total repos to scan                  
DEBU[0000] attempting to clone repo 1/2                  name="trufflehog - github" repo="https://github.com/bugbaba/test4truffleHog.git" source=SOURCE_TYPE_GITHUB
{"level":"debug","time":"2022-04-16T15:23:04+05:30","message":"executing: /usr/bin/git -C /tmp/trufflehog3964470179 log -p -U0 --full-history --all"}
DEBU[0000] Scanning complete. Scan time: 0.060681       
DEBU[0000] scanned 1/2 repos                            
2022/04/16 15:23:04 [updater parent] checking for updates...
FATA[0001] could open repo:                              error="repository does not exist"
2022/04/16 15:23:04 [updater parent] prog returned error: exit status 1
2022/04/16 15:23:04 [updater parent] prog exited with 1
```

**Screenshots**
Same repository working failing with `--json-legacy` flag
![image](https://user-images.githubusercontent.com/29960776/163670810-2f9ae219-a35a-480d-bbee-fde778746fef.png)

Same repository working with `--json` flag
![image](https://user-images.githubusercontent.com/29960776/163670820-947b9f29-4df2-4434-91c1-9977649e42f7.png)

**Desktop:**
 - OS: Linux Manjaro

**Comment by ahrav** [comment]
@bugbaba this should now be resolved with the recent release version 3.4.2.

**Comment by bugbaba** [comment]
Thank you.
